Resources/Prototypes/Body/Organs/human.yml

```
@@ -50,6 +50,8 @@
   - type: Examiner
   - type: BlockMovement
   - type: BadFood
+  - type: Food
+    requiredStomachs: 20 # DeltaV: Prevent eating brains as borgs don't remember what happened to them
```

the comment should be on line 53 not 54 because we're adding the component rather than changing a value
```suggestion
  - type: Food # DeltaV: Prevent eating brains as borgs don't remember what happened to them
    requiredStomachs: 20 
```

---

real